cmd/setup/setup.go

```diff
@@ -303,7 +359,8 @@ func mustExecuteMigration(ctx context.Context, eventstoreClient *eventstore.Even
 			"hint", pgErr.Hint,
 		)
 	}
-	logging.WithFields(logFields...).WithError(err).Fatal(errorMsg)
+	logging.WithFields(logFields...).WithError(err)
```

Comment:
this entry won't be logged because the level and message are not defined.
```suggestion
	logging.WithFields(logFields...).WithError(err).Error(errorMsg)
```